docker/azure-services/local-webapp-env-template

```diff
@@ -1,8 +1,6 @@
 # WARNING - All environment variables MUST be enclosed in double quotes to ensure the maximum degree of protection
 # against secrets being committed to version control unintentionally.
 export SERVER_PORT="3000"
-export REDIS_HOST="redis"
-export REDIS_PORT="6379"
 export SESSION_COOKIE_PASSWORD="the-password-must-be-at-least-32-characters-long"
```

Comment:
Could you please add below in this file?
```
CACHE_MAX_BYTE_SIZE, CACHE_CLEANUP_INT_MIN_SEC, CACHE_CLONE_BUFFER_ON
```

Reply:
These are not required as it will fall back to defaults.